manchester_traffic_offences/templates/plea/review.html

```diff
@@ -327,19 +294,49 @@
                 </tr>
                 <tr>
                     <th class="label">{% trans "Other contributors to household bills" %}</th>
-                    <td>{% if your_expenses.other_bill_payers %}{% trans "Yes" %}{% else %}{% trans "No" %}{% endif %}</td>
+                    <td>{{ your_expenses.other_bill_payers|yesno:_("Yes,No") }}</td>
                 </tr>
                 <tr>
                     <th class="label">{% trans "Total household expenses" %}</th>
-                    <td>£{{ your_expenses.total_household_expenses|intcomma }}</td>
+                    <td>£{{ your_expenses.total_household_expenses|floatformat:2|intcomma }}</td>
                 </tr>
                 <tr>
                     <th class="label">{% trans "Total other expenses" %}</th>
-                    <td>£{{ your_expenses.total_other_expenses|intcomma }}</td>
+                    <td>£{{ your_expenses.total_other_expenses|floatformat:2|intcomma }}</td>
                 </tr>
                 <tr>
                     <th class="label">{% trans "Total expenses" %}</th>
-                    <td>£{{ your_expenses.total_expenses|intcomma }}</td>
+                    <td>£{{ your_expenses.total_expenses|floatformat:2|intcomma }}</td>
+                </tr>
+            </tbody>
+        </table>
+        {% endif %}
+    {% endif %}
+{% else %}
```

Comment:
Can you change this this to check plea_made_by ==  "Company rep" instead of using else, it's dependent on the company journey rather than being a default so an explicit check will make this a bit easier to extend in the future.